core/src/View/Kokkos_ViewAlloc.hpp

```diff
@@ -242,6 +242,63 @@ struct ViewValueFunctorSequentialHostInit {
   }
 };
 
+template <class ElementType, class MemorySpace, class ExecutionSpace,
+          bool AllowPadding, bool Initialize, bool SequentialInit>
+Kokkos::Impl::SharedAllocationRecord<void, void>* make_shared_allocation_record(
+    const size_t& required_span_size, std::string_view label,
+    const MemorySpace& memory_space, const ExecutionSpace* exec_space,
+    std::integral_constant<bool, AllowPadding>,
```

Comment:
This template parameter appears to be unused. Ar you intending to do something with it in the future?

Reply:
Yeah we don't really need it here since we just pass the required_span_size in. That already includes padding if any.